server/sonar-server/src/main/java/org/sonar/server/user/AbstractUserSession.java

```diff
@@ -69,30 +68,21 @@ public UserSession checkOrganizationPermission(String organizationUuid, String p
   }
 
   @Override
-  public UserSession checkGlobalPermission(String globalPermission) {
-    return checkPermission(globalPermission);
-  }
-
-  @Override
-  public UserSession checkAnyPermissions(Collection<String> globalPermissionsToTest) {
-    List<String> userGlobalPermissions = globalPermissions();
-    for (String userGlobalPermission : userGlobalPermissions) {
-      if (globalPermissionsToTest.contains(userGlobalPermission)) {
-        return this;
-      }
-    }
-
-    throw new ForbiddenException(INSUFFICIENT_PRIVILEGES_MESSAGE);
+  public boolean hasPermission(String globalPermission) {
+    return isRoot() || globalPermissions().contains(globalPermission);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "globalPermissions"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)